src/main/java/net/wurstclient/hacks/AutoPlaceHack.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2014-2024 Wurst-Imperium and contributors.
+ *
+ * This source code is subject to the terms of the GNU General Public
+ * License, version 3. If a copy of the GPL was not distributed with this
+ * file, You can obtain one at: https://www.gnu.org/licenses/gpl-3.0.txt
+ */
+package net.wurstclient.hacks;
+
+import net.minecraft.client.gui.screen.ingame.HandledScreen;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.item.BlockItem;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.hit.HitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.wurstclient.Category;
+import net.wurstclient.SearchTags;
+import net.wurstclient.events.UpdateListener;
+import net.wurstclient.hack.Hack;
+import net.wurstclient.settings.CheckboxSetting;
+import net.wurstclient.util.BlockUtils;
+
+@SearchTags({"bridge", "auto place"})
+public final class AutoPlaceHack extends Hack implements UpdateListener
+{
+	private final CheckboxSetting onlyBelowFeet =
+		new CheckboxSetting("Only Below Feet",
+			"Only place blocks when target block is 1 block below your feet.",
+			false);
+	private final CheckboxSetting fastPlace =
+		new CheckboxSetting("Always FastPlace",
+			"Builds as if FastPlace was enabled, even if it's not.", true);
+	
+	public AutoPlaceHack()
+	{
+		super("AutoPlace");
+		setCategory(Category.BLOCKS);
+		addSetting(onlyBelowFeet);
+		addSetting(fastPlace);
+	}
+	
+	@Override
+	public void onEnable()
+	{
+		EVENTS.add(UpdateListener.class, this);
+	}
+	
+	@Override
+	public void onDisable()
+	{
+		EVENTS.remove(UpdateListener.class, this);
+	}
+	
+	@Override
+	public void onUpdate()
+	{
+		if(MC.currentScreen instanceof HandledScreen)
+			return;
+		
+		if(MC.itemUseCooldown != 0 && !fastPlace.isChecked())
+			return;

```

Comment:
This shouldn't be an issue since the AutoPlace hack itself ignores the value of `itemUseCooldown`. Setting it to 4 regardless of the `fastPlace` setting has the benefit of delaying manual block placing, so AutoPlace's placing speed is increased while everything else stays on the normal 4-tick cycle.